client.go

```diff
@@ -223,6 +225,8 @@ func (c client) SendAppMetrics(m ContainerMetric) error {
 		loggregator.WithGaugeValue("disk", float64(m.DiskBytes), "bytes"),
 		loggregator.WithGaugeValue("memory_quota", float64(m.MemoryBytesQuota), "bytes"),
 		loggregator.WithGaugeValue("disk_quota", float64(m.DiskBytesQuota), "bytes"),
+		loggregator.WithGaugeValue("rx_bytes", float64(m.RxBytes), "bytes"),
+		loggregator.WithGaugeValue("tx_bytes", float64(m.TxBytes), "bytes"),
```

Comment:
In the demo of this, it looked like rx_bytes + tx_bytes were cumulative counters (value increasing over the life of the interface/container), rather than gauges (point-in-time values of usage/consumption). 

Does it make sense to submit these via loggregator's EmitCounter logic instead?

Reply:
On the metrics side we prefer modeling everything as counters instead of gauges when possible. When using a counter a lost datapoint results in lower precision, while when using a gauge a lost datapoint is lost forever.

Specifically in this case it's often useful to know how much data a container has transmitted over a given window, which counters make possible.

Reply:
Very good points from both of you! 

We've changed the `tx_bytes` and `rx_bytes` to be sent as counters 👍 . See https://github.com/cloudfoundry/diego-logging-client/pull/82/files#diff-4b667feae66c9d46b21b9ecc19e8958cf4472d162ce0a47ac3e8386af8bbd8cfR246-R253